server/controller/login.js

```diff
@@ -0,0 +1,20 @@
+const bcrypt = require('bcryptjs');
+const { getuser } = require('../database/queries/getuser');
+const gentoken = require('./gentoken');
+
+const login = (req, response) => {
+  const { email, password } = req.body;
+  getuser(email).then((res) => {
+    if (res.rows.length > 0) {
+      bcrypt.compare(password, res.rows[0].password).then((result) => {
+        if (result) {
+          gentoken(email).then((token) => {
+            response.cookie('email', token, { maxAge: 60 * 60 * 60 });
+            response.status(200).json({ msg: 'done' });
+          });
+        } else response.json({ msg: 'password error' });
```

Comment:
@Jamalat-shamallakh These error responses would be easier for the client to handle if they were status code 4xx (for example, here a 401 might be appropriate).